### ISSUE TITLE
State Timeline: Support aligning the series name on the left or right

### ISSUE THREAD

**Issue by dylannorthrup** [grafana/grafana#94344]
This request is to make the alignment of the series name configurable, so a user can choose to line up the first letter of all series names. It is a repeat of Discussion [48659](https://github.com/grafana/grafana/discussions/48659) which was spawned from a [community post](https://community.grafana.com/t/how-format-text-for-metric-in-state-timeline/65049/2). I am creating this issue because it seems to have been dropped in those migrations.

**Why is this needed**:

Being able to left-align text makes it more readable for readers of languages read left to right. The current, unchangeable default of right-aligned text is not as conducive to "left-to-right" readability.

**What would you like to be added**:

An option for the State Timeline visualization to modify the alignment of the metric name.

**Who is this feature for?**

Dashboard creators and readers.

**Comment by septatrix** [comment]
I agree. Right-aligned text is only useful for RTL languages and numerical values. For normal text it makes it very annoying to read if they are anything but left aligned.

**Comment by leeoniya** [comment]
next release of uPlot will have what we need to implement this.

https://github.com/leeoniya/uPlot/commit/63a9b4a233fa2d4d2b334471fdb3487398a5e648
